Makefile

```diff
@@ -11,14 +11,21 @@ LDFLAGS=-ldflags "-X=github.com/peak/s5cmd/version.Version=$(VERSION) -X=github.
 build:
 	@go build ${GCFLAGS} ${LDFLAGS} -mod=vendor .
 
-RACE_FLAG := -race
+TEST_TYPE:=test_with_race
 ifeq ($(OS),Windows_NT)
-	RACE_FLAG =
+	TEST_TYPE=test_without_race
 endif
 
 .PHONY: test
-test:
-	@go test -mod=vendor -count=1 ${RACE_FLAG} ./...
+test: $(TEST_TYPE)
+
+.PHONY: test_with_race
+test_with_race:
+	@S5CMD_TEST_DISABLE_RACE=0 go test -mod=vendor -count=1 -race ./...
```

Comment:
> @S5CMD_TEST_DISABLE_RACE=0 go test -mod=vendor -count=1 -race ./...

Environment variable name is misleading. We build the test binary with race disabled, while running the tests in parallel.

Let's change the variable name to `S5CMD_BUILD_BINARY_WITHOUT_RACE_FLAG`.